app/src/main/res/drawable/sl_button_outlined_big_background.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@drawable/button_outlined_big_background"
+        android:state_pressed="false" />
```

Comment:
please remove state_pressed="false".
please check if items order is important is selectors.